presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationCompiler.java

```diff
@@ -254,4 +301,138 @@ private static Method getCombineFunction(Class<?> clazz, Class<?> stateClass)
 
         return stateClasses;
     }
+
+    private static String getDescription(AnnotatedElement base, AnnotatedElement override)
+    {
+        Description description = override.getAnnotation(Description.class);
+        if (description != null) {
+            return description.value();
+        }
+        description = base.getAnnotation(Description.class);
+        return (description == null) ? null : description.value();
+    }
+
+    private static Set<String> getLiteralParameter(Method inputFunction)
```

Comment:
I think we have that in to many places to successfully refactor this as part of other task.